release/release.py

```diff
@@ -245,8 +248,8 @@ def prereq(name, soft_check):
             fail(f"Pre-requisite not met: {name}. Error: {e}")
     prereq('Apache Maven CLI (mvn) in PATH', lambda: "Apache Maven" in execute("mvn -v"))
     prereq("svn CLI in PATH", lambda: "svn" in execute("svn --version"))
-    prereq("Verifying that you have no unstaged git changes", lambda: git.has_unstaged_changes())
-    prereq("Verifying that you have no staged git changes", lambda: git.has_staged_changes())
+    prereq("Verifying that you have no unstaged git changes", lambda: not git.has_unstaged_changes())
```

Comment:
@clolov Did you encounter this issue during the 4.0.1 release?

Reply:
@mimaison FYI 

This issue is straightforward, but I'm not sure why it wasn't exposed during the recent release